app/views/articles/index.html.slim

```diff
@@ -0,0 +1,8 @@
+h1 Articles
+ul
+  - articles.each do |article|
+   li
+      = link_to "#{article.user.firstname} : #{article.title}", article
```

Comment:
I think you can use decorator here to avoid adding additional logic in view and avoid duplicatable code